src/ansi-c/parser.y

```diff
@@ -3271,6 +3271,12 @@ cprover_function_contract_sequence_opt:
         | cprover_function_contract_sequence
         ;
 
+cprover_loop_contract:
+          cprover_contract_assigns_opt
+          loop_invariant_opt
```

Comment:
Could you please rename `loop_invariant_opt` to `cprover_contract_invariant_opt` for consistency? 

And, similarly, `cprover_contract_decreases_opt` when you resolve merge conflicts.

In the future, I would update the invariant keyword to `__CPROVER_invariant`, and we can specify both decreases and invariants in contracts for (recursive) functions as well.

Reply:
Renamed, and merge conflicts resolved.